jenkins/scripts/integration_test.sh

```diff
@@ -121,6 +123,24 @@ fi
 TEST_EXECUTER_IP="$(openstack port show -f json "${TEST_EXECUTER_PORT_NAME}" \
   | jq -r '.fixed_ips[0].ip_address')"
 
+if [[ "$OS_REGION_NAME" == "Fra1" ]]
+then 
+  # Fetch Free floating IP 
+  FLOATING_IP="$(openstack floating ip list --status DOWN -c "Floating IP Address" -f value | head -1 )"
+
+  if [[ -z "$FLOATING_IP" ]]
+  then
+    echo "No floating IP is available"
+    exit 1
+  fi
+  TEST_EXECUTER_IP="$FLOATING_IP"
+  
+  # Attach floating IP
+  openstack server add floating ip \
+  "${TEST_EXECUTER_VM_NAME}" \
+  "$FLOATING_IP"
+fi
+
```

Comment:
some explanation on this addition would help to understand this part

Reply:
I think this addition solves the issue that a Jenkins worker in Kna1 region cannot connect to VMs in Fra1 because of the different subnets. Therefore, we need to add floating IP to the VM in Fra1, so the worker in Kna1 can talk to them.

Reply:
thank you @kashifest